src/augmented_block.jl

```diff
@@ -90,6 +90,20 @@ function update_dual!(nlp::AugmentedNLPBlockModel, λ::AbstractVector)
     nlp.λ .= λ
 end
 
+"""
+    update_ρ!(
```

Comment:
that's something catchy with Julia: unicode is very handy to name variables/functions, but can be difficult to handle on specific terminal emulators (these poor people using windows) or to new users (it can be difficult to figure out that `rho` writes `\rho+ENTER` in Julia). 

As a general rule of thumb, I think we should avoid unicode in the name of the functions exposed in the API (but we are free to use it in the code hidden to the user otherwise). I would suggest renaming this function `update_rho!`